internal/clients/sources/sources_api.go

```diff
@@ -0,0 +1,15 @@
+package sources
+
+import (
+	"context"
+
+	"github.com/RHEnVision/provisioning-backend/internal/config"
+)
+
+func getSourcesAPIClient(ctx context.Context) (APIClient, error) {
+	return NewClientWithResponses(config.Sources.URL)
+}
+
+func init() {
+	GetSourcesAPIClient = getSourcesAPIClient
+}
```

Comment:
Not so visible, but this file has been renamed and methods removed, these methods are already in sources_client.

Reply:
Did you change the name of sources_impl to sources_api?

Reply:
Yeah, exactly ... impl sounded bit ambigous now that we have two implementations